<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="10" started-at="2014-04-13T23:19:25Z" finished-at="2014-04-13T23:19:25Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="10" started-at="2014-04-13T23:19:25Z" finished-at="2014-04-13T23:19:25Z">
      <class name="test.LoginTest">
        <test-method status="PASS" signature="initializeBrowser()[pri:0, instance:test.LoginTest@4b68cd47]" name="initializeBrowser" is-config="true" duration-ms="43848" started-at="2014-04-13T19:18:42Z" finished-at="2014-04-13T19:19:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeBrowser -->
        <test-method status="FAIL" signature="login(java.lang.String, java.lang.String)[pri:0, instance:test.LoginTest@4b68cd47]" name="login" duration-ms="0" started-at="2014-04-13T19:19:25Z" data-provider="getData" finished-at="2014-04-13T19:19:25Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Some DataProvider public static java.lang.Object[][] test.LoginTest.getData(java.lang.String) parameters unresolved:  at 1 type class java.lang.String
]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Some DataProvider public static java.lang.Object[][] test.LoginTest.getData(java.lang.String) parameters unresolved:  at 1 type class java.lang.String

	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:128)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:429)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- test.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
